packages/office-ui-fabric-react/src/components/Panel/Panel.styles.ts

```diff
@@ -286,7 +286,7 @@ export const getStyles = (props: IPanelStyleProps): IPanelStyles => {
       classNames.navigation,
       {
         display: 'flex',
-        justifyContent: 'flex-end'
+        alignItems: 'flex-end'
```

Comment:
wait, what's the intention here? why is this needed?
isn't there a margin property that needs to get removed?

Reply:
we likely want justify-content:space-between to force the title and X button to be on the edges

Reply:
I just tried out space-between successfully

Reply:
same tested it on IE 11, chrome, and edge again all work